sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Extraneous <code>9</code>  output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>No, this is simply printing the start step at which python profiling was started (step 9).</p><p> </p>